### ISSUE TITLE
graphql linting not working

### ISSUE THREAD

**Issue by sulliwane** [apollographql/eslint-plugin-graphql#207]
I have this `.graphqlconfig` at the folder root:
```json
{
  "projects": {
    "project": {
      "schemaPath": "schema.graphql",
      "extensions": {
        "endpoints": {
          "default": {
            "url": "http://X.X.X.X:8080",
            "headers": {
              "X-Access-Key": "SECRET"
            },
            "subscription": "ws://X.X.X.X:8080"
          }
        }
      }
    }
  }
}

```
I ran `graphql get-schema` that downloaded `schema.graphql` locally.
I ran `npm install --save-dev eslint-plugin-graphql`
Finally, here is my `.eslintrc`:
```json
{
  "rules": {
    "graphql/template-strings": ["error", { "env": "literal" }]
  },
  "plugins": ["graphql"]
}

```
But still, I don't get any linting warning/errors even though I'm using wrong struct names, ex:
```js
const gql = require('graphql-tag');
const query = gql`
  query {
    notExists {
      meNeither
    }
  }
`;
```

Any idea? Many thanks.

Note: Atom v1.33.1

**Comment by jnwng** [comment]
@sulliwane the environment you have configured (`literal`) is meant for files ending in `.gql/.graphql`, not the "template literal". by default, the `apollo` environment should cover your needs here. see https://github.com/apollographql/eslint-plugin-graphql#graphql-literal-files

please re-open if that still doesn't work!
